fhir-server/src/main/java/org/linuxforhealth/fhir/server/filter/rest/FHIRRestServletFilter.java

```diff
@@ -61,6 +62,8 @@ public class FHIRRestServletFilter extends HttpFilter {
     private static final String preferHeaderName = "Prefer";
     private static final String preferHandlingHeaderSectionName = "handling";
     private static final String preferReturnHeaderSectionName = "return";
+    
+    private static List<String> configuredTenants = null;
```

Comment:
Can we make this a set instead?

Reply:
This has been fixed as suggested to unmodifiable concurrent hashmap.